WebCrawler/Model/Page.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace WebCrawler
+{
+    /// <summary>
+    /// Page class
+    /// </summary>
+    public class Page
+    {
+        #region Constructor
+
+        /// <summary>
+        /// Default constructor.
+        /// </summary>
+        public Page() { }
+
+        #endregion
+        #region Private Instance Fields
+
+        private int _size;
+        private string _text;
+        private string _url;
+        private int _viewstateSize;
+
+        #endregion
+        #region Public Properties
+
+        public int Size
+        {
+            get { return _size; }
+        }
+
+        public string Text
+        {
+            get { return _text; }
+            set
+            {
+                _text = value;
+                _size = value.Length;
+            }
+        }
+
+        public string Url
+        {
+            get { return _url; }
+            set { _url = value; }
+        }
+        #endregion
+    }
+}
```

Comment:
Use [Auto Implemented Properties ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties#auto-implemented-properties) to make it nicer and easier to change